annotation-client/src/test/java/eu/europeana/annotation/client/integration/webanno/link/BaseLinkingTest.java

```diff
@@ -18,6 +20,12 @@ protected Annotation parseLink(String jsonString) throws JsonParseException {
 		MotivationTypes motivationType = MotivationTypes.LINKING;
 		return parseAnnotation(jsonString, motivationType);		
 	}
+	
+    @Test
+    public void createLinkAnnotation() throws JsonParseException, IOException {
```

Comment:
please add a test also for the other format where the body contains the id

Reply:
Done.